src/protagonist/Orchestrator.Tests/Integration/ManifestHandlingTests.cs

```diff
@@ -404,6 +459,51 @@ public async Task Get_V3ManifestForImage_ReturnsManifestNoThumbnails_WhenNoThumb
         response.Headers.CacheControl.MaxAge.Should().BeGreaterThan(TimeSpan.FromSeconds(2));
     }
     
+    [Fact]
+    public async Task Get_V3ManifestForImage_ReturnsManifestWithImageServices_WhenImageChannelSet()
+    {
+        // Arrange
+        var id = AssetIdGenerator.GetAssetId();
+        await dbFixture.DbContext.Images.AddTestAsset(id, origin: "testorigin")
+            .WithTestDeliveryChannel(AssetDeliveryChannels.Image);
+        await dbFixture.DbContext.SaveChangesAsync();
+            
+        var path = $"iiif-manifest/{id}";
+
+        // Act
+        var response = await httpClient.GetAsync(path);
+            
+        // Assert
+        var jsonResponse = JObject.Parse(await response.Content.ReadAsStringAsync());
+        jsonResponse["id"].ToString().Should().Be($"http://localhost/iiif-manifest/{id}");
+        jsonResponse.SelectToken("items[0].items[0].items[0].body").Should().NotBeNull();

```

Comment:
FYI (not worth changing but just to be aware) some of these test pre-date the iiif-net nuget package handling deserialisation correctly which is why `SelectToken` was used (it can still be the best choice sometimes though!). Now that it is supported you can deserialize to a strongly typed model if required.

```cs
// this
var jsonResponse = JObject.Parse(await response.Content.ReadAsStringAsync());
jsonResponse["id"].ToString().Should().Be($"http://localhost/iiif-manifest/{id}");

// can be
var manifest = (await response.Content.ReadAsStreamAsync()).FromJsonStream<IIIF3.Manifest>();
manifest.Id.Should().Be($"http://localhost/iiif-manifest/{id}");